test-procedure-guideline.md

```diff
@@ -0,0 +1,35 @@
+# How to test the app
+
+## Where to do the test?
+When you test the app, please make sure that you test it:
+- on mobile
+- on desktop
+- On different browsers (firefox, chrome, edge, safari) => these 4 are representing 94% of the users
+
+## how to conduct the test?
+Here are a few step that you should follow:
+1. Create a new comic with 1 panel
+2. add a text and modify its content and its font size
+3. Add and modify multiple assets (rotate, miror, resize)
+   - try to use all the possibilities on different assets
+   - try to use assets that you never used
+   - put an asset out of the panel
+   - use different categories and subcategory of the catalog
+   - Do a search with the catalog
+   - Filter results
+   - delete an asset
+4. Add an asset without modification
+5. move some asset up and down using the buttons
+6. move some asset up and down using the layers popup
+7. Look at the preview
+8. Open the download page
+9. Download your comic and verify that it looks as on the editor
+10. Save your comic as a draft
+11. Navigate back to the index
+12. Open the draft
+13. Modify the draft (redo points 2 to 6)
+14. Save the draft
+15. Navigate back to the index
+16. Delete the draft
+17. Create a new comic with more than 1 panel
+18. Redo points 2 to 16 (the assets should be on different panels)
```

Comment:
add as a last point to report issues on the PR - I mean it's kinda self explanatory but just for completion sake :)